### ISSUE TITLE
Loss Function： loss_wh

### ISSUE THREAD

**Issue by jinmallll** [experiencor/basic-yolo-keras#86]
pred_box_wh = tf.exp(y_pred[..., 2:4]) * np.reshape(self.anchors, [1,1,1,self.nb_box,2])
loss_wh    = tf.reduce_sum(tf.square(true_box_wh-pred_box_wh)     * coord_mask) / (nb_coord_box + 1e-6) / 2.
Will the anchors influence the gradients when calculating the loss function?

Will it be better like this in the loss_wh?
pred_box_wh = tf.exp(y_pred[..., 2:4])
true_box_wh = true_box_wh/np.reshape(self.anchors, [1,1,1,self.nb_box,2])
loss_wh    = tf.reduce_sum(tf.square(true_box_wh-pred_box_wh)     * coord_mask) / (nb_coord_box + 1e-6) / 2.

**Comment by experiencor** [comment]
@jinmallll I cannot say which one is better. One is a scaled version of the other.

**Comment by jinmallll** [comment]
OK. Thank you very much!
